doozerlib/dblib.py

```diff
@@ -327,12 +327,17 @@ def handle_missing_columns(self, payload, table_name):
             for column in missing_columns:
                 missing_column = column[0]
                 column_type = column[1][0]
-                cursor.execute(f"alter table {table_name} add column `{missing_column}` {column_type}")
-                self._table_column_cache[table_name][missing_column] = True
-
-                self.runtime.logger.info("Added new column [{}] of identified type [{}] to table [{}] "
-                                         "of database [{}].".format(missing_column, column_type,
-                                                                    table_name, self.db))
+                try:
+                    cursor.execute(f"alter table {table_name} add column `{missing_column}` {column_type}")
+                    self._table_column_cache[table_name][missing_column] = True
+
+                    self.runtime.logger.info("Added new column [{}] of identified type [{}] to table [{}] "
+                                             "of database [{}].".format(missing_column, column_type,
+                                                                        table_name, self.db))
+                except Exception as e:
+                    self.runtime.logger.warning("Could not add new column [{}] of identified type [{}] to table [{}] "
+                                                "of database [{}]. Error: {}".format(missing_column, column_type,
+                                                                                     table_name, self.db, e))
```

Comment:
at this point, having identified a column that we can't write, can we remove that column from the payload so that the rest of it can still be written? i'm expecting that the next thing you get after this warning is a SQL error trying to write to an undefined column.

Reply:
Actually there's no error raised, just a Null value is entered in the DB, same as if we remove it from the payload. That's why it slipped my mind. But its better to delete it from the payload like you said.